JASP-Engine/JASP/R/auditClassicalBenfordsLaw.R

```diff
@@ -254,40 +254,39 @@ auditClassicalBenfordsLaw <- function(jaspResults, dataset, options, ...){
   if(!is.null(benfordsLawContainer[["benfordsLawTestTable"]])) 
     return()
 
-  tableTitle <- paste0("<b>Table ", 
-                    jaspResults[["tabNumber"]]$object, 
-                    ".</b> Goodness-of-fit Test")
+  tableTitle <- gettextf("<b>Table %i.</b> Goodness-of-fit Test", 
+                          jaspResults[["tabNumber"]]$object)
```

Comment:
%i can only be used if you are sure for 100% that the substituted value is indeed an integer. Else it crashes... %s is better if the type is unsure.